rules/android/android_library.bzl

```diff
@@ -64,7 +64,7 @@ def android_library(
             resources = [file for file in resource_files if file.endswith(".xml")],
             manifest = attrs.get("manifest"),
             baseline = lint_baseline,
-            lint_config = lint_options.get("lint_config", None),
+            lint_config = lint_options.get("config", None),
```

Comment:
We will change on grazel too?

Reply:
not really this just to be align with grazel